lib/hypertrack.dart

```diff
@@ -1,23 +1,25 @@
 import 'package:flutter/services.dart';
-import 'package:hypertrack_plugin/hypertrack_platform_interface.dart';
+import 'package:hypertrack_plugin/src/sdk_wrapper.dart';
 
-import 'const/constants.dart';
+import 'data_types/availability.dart';
+import 'data_types/location.dart';
+import 'data_types/tracking_state.dart';
 
-/// Plugin allows you to use your application as a location data source
-/// feeding HyperTrack platform.
+/// This plugin allows you to use Hypertrack SDK for Flutter apps to get realtime device location
 class HyperTrack {
-  final HypertrackPlatformInterface _pluginInterface =
-      HypertrackPlatformInterface.instance;
+  HyperTrack._();
```

Comment:
what does this do?

Reply:
>MyClass._(): There's nothing special about this; it's the same as MyClass._private() but with a different name. It's not a default constructor, and it's still a private, named constructor (i.e., callers must use MyClass._() to invoke it). It's a case where the class author wanted a private constructor but didn't feel like choosing a name. (Naming things is hard.)
ref.: https://stackoverflow.com/questions/57812052/the-difference-between-the-use-of-constructor-classname-and-classname